helm/cluster-api-provider-aws/templates/apps_v1_deployment_capa-controller-manager.yaml

```diff
@@ -50,7 +50,8 @@ spec:
         env:
         - name: AWS_SHARED_CREDENTIALS_FILE
           value: /home/.aws/credentials
-        image: '{{ .Values.infrastructure.image.name }}:{{ .Values.tag }}'
+        image: '{{ .Values.infrastructure.image.domain }}/{{ .Values.infrastructure.image.name
+          }}:{{ .Values.tag }}'
```

Comment:
This is wrong. There is a bug on kustomize when lines are too long which adds an extra line

Reply:
To fix the issue, try removing the spaces around `{{` and `}}`

Reply:
https://github.com/giantswarm/cluster-api-provider-aws-app/compare/fix_line_wrap?expand=1